src/web/mjs/connectors/Qiman5.mjs

```diff
@@ -53,6 +53,19 @@ export default class Qiman5 extends SinMH {
         `;
     }
 
+    canHandleURI(uri) {
+        super.initialize()
+            .then(() => {
+                return uri.href.includes(this.url);
+            });
+    }
+
+    async _initializeConnector() {
+        const request = new Request(this.url, this.requestOptions);
+        this.url = (await this.fetchDOM(request, 'a#href')).pop().href;
```

Comment:
This may be error prone in case the last found `anchor` element is a cross reference to a different domain.

Reply:
Aslo shouldn't it use `.origin` instead of `.href`?

Reply:
1) There is only one 'a' element on the page, and its 'id' is 'href'. There shouldnt be more than one 'a' node with the same 'id' 🤷 .  And since its a portal page, its a given that it will be to another domain, i dont get it.

2) True, origin is more error resistant.